Inclusões e alterações em Adobe (#2307)
mercadopago/devsite-docs: guides/adobe-commerce/installation-magento-marketplace.en.md

# Installation via Adobe Commerce Marketplace

You can install the Mercado Pago extension at no cost through the Adobe Commerce store, it's very simple. Follow these steps:

> 1. Access the [Official Adobe Commerce Store](https://marketplace.magento.com/). In the search engine, you must enter Mercado Pago and select the [official extension](https://commercemarketplace.adobe.com/mercadopago-adb-payment.html). 

Reviewer: ```suggestion
1. Access the [Official Adobe Commerce Store](https://marketplace.magento.com/). In the search engine, type Mercado Pago and select the [official extension](https://commercemarketplace.adobe.com/mercadopago-adb-payment.html). 
```
